.drone.yml

```diff
@@ -172,6 +172,57 @@ trigger:
   - refs/tags/**
   - refs/pull/**
 
+
+---
+kind: pipeline
+type: vm
+name: windows-ltsc2022
```

Comment:
Just making a note that the official docker images that provide 2022 images have this as their suffix so this is good to go.